apps/condo/domains/news/constants/newsCategory.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Generated by `createschema news.OrganizationNewsItem 'organization:Relationship:Organization:CASCADE; title:Text; body:Text; type:Select:common,emergency'`
+ */
+
+const NEWS_CATEGORY_WATER = 'water'
+const NEWS_CATEGORY_HEATING = 'heating'
+const NEWS_CATEGORY_ELECTRICITY = 'electricity'
+const NEWS_CATEGORY_ALL_UTILITIES = 'allUtilities'
+const NEWS_CATEGORY_ELECTRICITY_AND_WATER = 'electricityAndWater'
+const NEWS_CATEGORY_LIGHT = 'light'
```

Comment:
We do not need Light category – please check categories from News Sharing Greendom